static/scripts/dataprivacy/paginated_form.js

```diff
@@ -70,6 +70,13 @@ function updateButton(selectedIndex) {
 		document.querySelector('.form #nextSection').removeAttribute('disabled');
 		document.querySelector('.form #nextSection').classList.remove('hidden');
 	}
+
+	if (selectedIndex === 1) {
+		document.querySelector('.form #firstSection').classList.remove('hidden');
+	} else {
+		document.querySelector('.form #firstSection').classList.add('hidden');
+	}
```

Comment:
please adjust the selector here as well.
Even better: Move the querySelector part into a variable and check if the selector even matches any element.
```js
	const btnLogout = document.querySelector('.form #btn-logout');
	if(btnLogout){
		btnLogout.classList[selectedIndex === 1 ? "remove" : "add"]('hidden');
	}
```